src/main/java/bio/terra/common/tracing/TracingHook.java

```diff
@@ -0,0 +1,214 @@
+package bio.terra.common.tracing;
+
+import bio.terra.stairway.*;
```

Comment:
Not sure if we've been strict about this at all, but Google's style guide disallows wildcard imports ([ref](https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports)). CH was careful about this in the AoU repo... I'm ambivalent, especially since IDEs seem to reach for wildcards sometimes.

Reply:
I just setup it correctly when I onboard AoU. We missed Google-Java-Format in our intellij. 
Instruction: https://docs.google.com/document/d/1DtESBapEzvuti7xODTFPHorwmLM7LybF-6D5lhbIkLU/edit#heading=h.vj6yd

Reply:
There's good reasons to not use wildcard imports, but I'm not really interested in fighting my IDE much less everyone else's IDE in code review. IF we have a hard desire to stop wildcard imports and can get everyone to change their IDE settings/have a pre-submit to disallow them so they don't get submitted, I will happily jump on board. But without broad consensus/enforcement, I'm not interested in trying.

Reply:
It is a trivial IntelliJ configuration to stop getting wildcard imports. That was the norm for TDR and I think it should be the norm here as well. With that config in place, we did not need any pre-submit rule.